core/objectUtils.js

```diff
@@ -39,11 +39,18 @@ export const isPropTrue = (prop) => R.pipe(getProp(prop), R.equals(true))
 
 export const getParentUuid = R.propOr(null, keys.parentUuid)
 
+const _getText = ({ propKey, lang }) => R.pipe(getProp(propKey, {}), R.prop(lang))
+const _getTextWithDefault = ({ propKey, lang, defaultLang }) => (obj) => {
+  const text = _getText({ propKey, lang })(obj)
+  return !text && Boolean(defaultLang) && lang !== defaultLang ? _getText({ propKey, lang: defaultLang })(obj) : text
+}
 export const getLabels = getProp(keysProps.labels, {})
-export const getLabel = (lang, defaultTo = null) => R.pipe(getLabels, R.propOr(defaultTo, lang))
+export const getLabel = (lang, defaultLang = null) =>
```

Comment:
will this cause any backwards compatibility issue?

Reply:
I'll double check it... but it looks more correct to pass a default language instead of a default value, isn't it?